src/Plug.vala

```diff
@@ -113,6 +113,8 @@ public class MouseTouchpad.Plug : Switchboard.Plug {
         search_results.set ("%s → %s".printf (display_name, _("Long-press secondary click")), "general");
         search_results.set ("%s → %s".printf (display_name, _("Long-press length")), "general");
         search_results.set ("%s → %s".printf (display_name, _("Middle click paste")), "general");
```

Comment:
Looks like "Middle click paste" is listed twice, this might also be a good change to fix that?